# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/vr/service/VRServiceManager.cpp
# Commit: 1d9384906287
# Full Hash: 1d93849062870a8d492eeef2150ecea174cb6223
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2019-05-26 21:12:53
# Regressor Bug: 1550422
# File Overlap Count: 1
# Description:
#   Bug 1550422 - P15. Move Skip and Once gfxPrefs to StaticPrefs. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D31259
# ==============================================================================

diff -r f5ebdcca640d -r 1d9384906287 gfx/vr/service/VRServiceManager.cpp
--- a/gfx/vr/service/VRServiceManager.cpp	Sun May 26 14:29:56 2019 +0000
+++ b/gfx/vr/service/VRServiceManager.cpp	Sun May 26 14:30:14 2019 +0000
@@ -7,6 +7,7 @@
 #include "VRServiceManager.h"
 #include "VRGPUChild.h"
 #include "mozilla/gfx/GPUParent.h"
+#include "mozilla/StaticPrefs.h"
 
 namespace mozilla {
 namespace gfx {
@@ -45,7 +46,7 @@
     }
     VRGPUChild::Shutdown();
   }
-  if (gfxPrefs::VRProcessEnabled()) {
+  if (StaticPrefs::VRProcessEnabled()) {
     // Using PGPU channel to tell the main process
     // to shutdown VR process.
     gfx::GPUParent* gpu = GPUParent::GetSingleton();
@@ -55,7 +56,7 @@
 }
 
 void VRServiceManager::CreateService() {
-  if (!gfxPrefs::VRProcessEnabled()) {
+  if (!StaticPrefs::VRProcessEnabled()) {
     mVRService = VRService::Create();
   }
 }